haybaler.py

```diff
@@ -275,11 +275,14 @@ def main(input_files, input_path, output_path, output_file, readcount_limit, rpm
     excluded_taxa_df.fillna("no", inplace=True)
     excluded_taxa_df.to_csv(output_path + "/excluded_taxa.csv", sep="\t")  # save the df with the excluded taxa and the reason
     # when concating two df's, the value species gets lost, so it needs to be added afterwards
-    with open(output_path + "/excluded_taxa.csv", 'r+') as f:
-        content = f.read()
-        if not content.split()[0] == "species":
-            f.seek(0, 0)
-            f.write(f"species" + content)
+    try:
+        with open(output_path + "/excluded_taxa.csv", 'r+') as f:
+            content = f.read()
+            if not content.split()[0] == "species":
+                f.seek(0, 0)
+                f.write(f"species" + content)
+    except FileNotFoundError:
+        print("WARNING: File excluded_taxa.csv can't be read. Please check if it was created and if species column exists.")
```

Comment:
Nice. Output file can't be created and written is more clear, don't you think ?

Reply:
I wasn't sure if there is any way, that a file was created and maybe misplaced or misnamed so that it can' t be opened. 

But if this is totaly impossible, your suggested warning is definitely more clear.

Reply:
Ok, please edit then (I can't edit your PR I think) then merge. Pushing dot (.) on the keyboard should get you the web editor. Cheers.